buildprogram.bat

```diff
@@ -1,8 +1,13 @@
 
 set mypath=%~dp0
-pyinstaller --onefile resources\twitchcommands.py --add-data "C:\Users\%USERNAME%\AppData\Local\Programs\Python\Python310\Lib\site-packages\ahk";ahk --icon resources\appicon.ico 
+python -c "import sysconfig; print(sysconfig.get_path('purelib') + '\\ahk')" > tmpFile
+set /p PYSITEPKG_DIR= < tmpFile
+DEL tmpFile
+pyinstaller --onefile resources\twitchcommands.py --add-data %PYSITEPKG_DIR%;ahk --icon resources\appicon.ico 
 move "%mypath%dist\twitchcommands.exe" "%mypath%/"
 move "%mypath%\twitchcommands.spec" "%mypath%/resources"
+DEL /S /Q "%mypath%OPENGOALTWITCHv0.1.backup.exe"
+RENAME "%mypath%OPENGOALTWITCHv0.1.exe" "OPENGOALTWITCHv0.1.backup.exe"
```

Comment:
it kept failing to overwrite the old exe for me, so I just moved it to a backup file